packages/reactive-element/src/reactive-element.ts

```diff
@@ -588,6 +601,7 @@ export abstract class ReactiveElement
   static get observedAttributes() {
```

Comment:
We don't store the attributes anywhere right now, and to do so would add to retained memory, so I'd like to be conservative there.  __collectMetadata is idempotent, so this should be safe. I would like to simplify how we track element properties eventually maybe even soon, but it might be separate from this?